boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/JAXRSBoosterPackConfigurator.java

```diff
@@ -5,9 +5,12 @@
 
 public class JAXRSBoosterPackConfigurator implements BoosterPackConfigurator {
 
+	//default to the EE8 feature
+	String featureGAV = "jaxrs-2.1";
```

Comment:
Wondering if we really want to default here? What happens when the next Jakarta EE version comes out? Do we leave this as the default, or change to the latest? Are we always calling setFeatureString method regardless of the booster version?

Reply:
yes we are calling it everytime, so i could get rid of this default value